include/deal.II/lac/vector.h

```diff
@@ -1327,8 +1275,6 @@ template <typename Number>
 inline void
 Vector<Number>::swap(Vector<Number> &v)
 {
-  std::swap(vec_size, v.vec_size);
-  std::swap(max_vec_size, v.max_vec_size);
   std::swap(values, v.values);
```

Comment:
This is another thing to look at in #7646: we should swap partitioners here too.